Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/RenderOptions.cpp

```diff
@@ -318,6 +316,9 @@ namespace EMStudio
 
         options.m_manipulatorMode = static_cast<ManipulatorMode>(settings->value("manipulatorMode", options.m_manipulatorMode).toInt());
 
+        AZ::Render::RenderActorSettings& renderActorSettings = EMotionFX::GetRenderActorSettings();
+        options.CopyToRenderActorSettings(renderActorSettings);
```

Comment:
Is this also being called when changing a color in the preferences window? Is that working? I couldn't see the on option changed callback being modified here.